components/trial_benefits_modal/trial_benefits_modal.tsx

```diff
@@ -176,6 +183,7 @@ const TrialBenefitsModal = ({
             />
         ),
         bottomLeftMessage: formatMessage({id: 'trial_benefits.modal.onlyVisibleToAdmins', defaultMessage: 'Only visible to admins'}),
+        isCloud: license?.Cloud === 'true',
```

Comment:
Could use `isCloud` const already defined in line 53.

Reply:
+1